packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md

```diff
@@ -1,4 +1,4 @@
-### Step 1: Create a sign-up link for new users
+### Step 1: Click sign-up link
```

Comment:
We agreed in the standup meeting where we discussed this that we'd remove the word "Step" from the step headings, didn't we? So for example this line would just be `### 1: Click sign-up link`

Reply:
"Step" text removed.